### PR TITLE
refactor(shared): RichTextEditor loading serializer asynchronously and use it in the blocks

### PR DESCRIPTION
**Pull request by celine-s**
* update to react 18: because [of this fondue release](https://github.com/Frontify/fondue/pull/1355/files#diff-5ab71f4254fac269ededf6e398bf8bae6ae00d9ca962820b3524bc9b3a6a784e). 
* fix breaking changes [these breaking changes](https://github.com/Frontify/fondue/pull/1308/files) should be fixed
* adapt to new serializer

### CODE REVIEW

File: packages/sketchfab-block/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "@babel/core": "7.20.2",
         "@frontify/eslint-config-react": "0.15.5",
         "@frontify/frontify-cli": "5.3.10",
-        "@types/react": "17.0.53",
+        "@types/react": "18.0.35",
         "@types/react-dom": "17.0.17",
```
Comment (ragi96) [inline packages/sketchfab-block/package.json:21]:
```suggestion
        "@types/react-dom": "18.2.1",
```

File: packages/storybook-block/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         "@babel/core": "7.20.2",
         "@frontify/eslint-config-react": "0.15.5",
         "@frontify/frontify-cli": "5.3.10",
-        "@types/react": "17.0.53",
+        "@types/react": "18.0.35",
         "@types/react-dom": "17.0.17",
```
Comment (ragi96) [inline packages/storybook-block/package.json:20]:
```suggestion
        "@types/react-dom": "18.2.1",
```

File: packages/text-block/package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
         "@frontify/eslint-config-react": "0.15.5",
         "@frontify/frontify-cli": "5.3.10",
         "@types/lodash-es": "4.17.6",
-        "@types/react": "17.0.53",
+        "@types/react": "18.0.35",
         "@types/react-dom": "17.0.17",
```
Comment (ragi96) [inline packages/text-block/package.json:22]:
```suggestion
        "@types/react-dom": "18.2.1",
```

**Comment by ragi96** [comment]
shared hooks adjust tests 
`Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot`

File: packages/color-scale-block/src/otherTests [hunk 4]
```diff
@@ -0,0 +1,110 @@
+const AltTextInput = '[data-test-id="alt-text-input"]';
```
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/color-scale-block/src/otherTests]: removed as its useless and contains ThumbnailGrid Tests

File: packages/checklist-block/src/components/FocusController.tsx [hunk 5]
```diff
@@ -23,7 +23,6 @@ export const FocusController: FC<FocusControllerProps> = ({ children, width = Fo
     const { isFocused, focusProps } = useFocusRing();
 
     const focusControllerRef = useRef<HTMLDivElement | null>(null);
-    const childRef = useRef<HTMLElement | null>(null);
```
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/checklist-block/src/components/FocusController.tsx:26]: Assumption was wrong, its needed for entering the rte via keyboard, so this needs to be fixed

[the remark this answers is not in the document]
Comment (ragi96) [reply packages/checklist-block/src/components/FocusController.tsx]: Fixed

**Review by ragi96 — APPROVED** [review]

**Review by fulopdaniel — APPROVED** [review]

**Pull request merged**